src/ctx.cpp

```diff
@@ -54,14 +54,19 @@ projCtx pj_get_ctx( projPJ pj )
 }
 
 /************************************************************************/
-/*                             pj_set_ctx()                             */
-/*                                                                      */
-/*      Note we do not deallocate the old context!                      */
+/*                        proj_assign_context()                         */
 /************************************************************************/
 
-void pj_set_ctx( projPJ pj, projCtx ctx )
-
+/** \brief Re-assign a context to a PJ* object.
+ *
+ * This may be useful if the PJ* has been created with a context that is
+ * thread-specific, and is later used in another thread. In that case,
+ * the user may want to assign another thread-specific context to the
+ * object.
+ */
+void proj_assign_context( PJ* pj, PJ_CONTEXT *ctx )
 {
+    // Note we do not deallocate the old context!
```

Comment:
this comment doesn't really make sense to me. Could be removed

Reply:
I don't understand it either, but it was in the old function description so I figured it might have some value. I'll remove it.